src/routes/console/project-[project]/functions/function-[function]/executions/+page.svelte

```diff
@@ -142,6 +166,52 @@
                         <TableCellText width={80} title="Duration">
                             {calculateTime(execution.duration)}
                         </TableCellText>
+                        <TableCell width={40} showOverflow>
+                            <DropList
+                                bind:show={showDropdown[index]}
+                                placement="bottom-start"
+                                noArrow>
+                                <button
+                                    class="button is-only-icon is-text"
+                                    aria-label="More options"
+                                    on:click|preventDefault={() => {
+                                        showDropdown[index] = !showDropdown[index];
+                                    }}>
+                                    <span class="icon-dots-horizontal" aria-hidden="true" />
+                                </button>
```

Comment:
Let's use the `Button` component for this :+1:

Reply:
done